jwtclient/cmd/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/dcos/dcos-go/jwtclient"
+)
+
+var (
+	flagURL       = flag.String("url", "", "URL to query")
+	flagIAMConfig = flag.String("iam-config", "", "Path to IAM config")
```

Comment:
I'm not keen on having flags in a library, these should be configured from the calling code, not from the command line.